Sources/NostrSDK/Events/NostrEvent.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 /// A structure that describes a Nostr event.
 ///
 /// > Note: [NIP-01 Specification](https://github.com/nostr-protocol/nips/blob/master/01.md#events-and-signatures)
-public class NostrEvent: Codable, Equatable, Hashable {
+public class NostrEvent: Codable, Equatable, Hashable, LabelTagInterpreting {
```

Comment:
Although labels can exist on any NostrEvent, I've chosen to separate that part of the class out into a separate protocol/extension so that it's easier to maintain, and provides a logical separation.